.gitignore

```diff
@@ -15,7 +16,6 @@ node_modules
 .idea
 *.sublime-project
 *.sublime-workspace
-.vscode
```

Comment:
This might be slightly more useful:

(ignore everything from .vscode but unignore these two files)

```
.vscode/**/*
!.vscode/launch.json
!.vscode/tasks.json
```

Reply:
b098104